src/modules/feature-modules/accessor/pages/innovations/innovations-review.component.ts

```diff
@@ -354,15 +355,16 @@ export class InnovationsReviewComponent extends CoreComponent implements OnInit
                 status: item.support.status,
                 updatedAt: item.support.updatedAt,
                 updatedBy: item.support.updatedBy,
-                closedReason: {
-                  value: item.support.closedReason,
+                closeReason: {
+                  value: item.support.closeReason,
                   label:
-                    item.support.closedReason === InnovationStatusEnum.ARCHIVED
+                    item.support.closeReason === InnovationSupportCloseReasonEnum.ARCHIVE
                       ? 'Archived'
-                      : item.support.closedReason === 'STOPPED_SHARED'
+                      : item.support.closeReason === InnovationSupportCloseReasonEnum.STOP_SHARE
                         ? 'Stopped sharing'
-                        : item.support.closedReason === InnovationSupportStatusEnum.CLOSED
-                          ? 'Closed'
+                        : item.support.closeReason === InnovationSupportCloseReasonEnum.SUPPORT_COMPLETE
+                          ? // TODO: REVISIT NAMING WHEN DESIGN IS PROVIDED
+                            'Closed'
```

Comment:
We could put this on `en.ts` it would simplify this logic.

Reply:
Good idea!